lib/worker.js

```diff
@@ -4,8 +4,16 @@ const path = require('path')
 const _ = require('lodash')
 const fs = require('fs')
 
-const getJSONConf = (env, type, path) => {
-  const conf = JSON.parse(fs.readFileSync(path, 'utf8'))
+const getJSONConf = (env, type, confPath) => {
```

Comment:
since this function is used only here, `confPath` contains a static filename, and we deconstruct `confPath` to reconstruct it back, I'd suggest sticking to one of the following approaches:
* `getJSONConf` accepts the valid path, that's been constructed by the caller
* `getJSONConf` accepts `serviceRoot` to construct the path and read the file

what do you think?

Reply:
Well first thought to use valid path from suggested approach 1, but then thought that maybe in future we would reuse this method also somewhere and then in each call the path should be constructed from outside so we would copy some resolving code possibly. That's why I've decided to construct env specific path inside the method so it would resolve path specific conf on fly.

using 2nd approach would require to force method only for accepting `/config/common.json` file I think if I understood you correctly

Reply:
>using 2nd approach would require to force method only for using /config/common.json file I think if I understood you correctly

yes, i suggested this based on the current behaviour